third-party/nats/README.md

```diff
@@ -0,0 +1,72 @@
+# @kubernetes-models/nats
+
+NATS models.
+
+This package contains the following operators:
+
+- [NATS Controllers for Kubernetes (NACK)](https://github.com/nats-io/nack)
+
+Feel free to submit an issue or a pull request if the operator you are using is not here.
+
+## Installation
+
+Install with npm.
+
+```sh
+npm install @kubernetes-models/nats
+```
+
+## Usage
+
+### NATS Controllers for Kubernetes (NACK)
+
+```js
+import { Stream } from "@kubernetes-models/nats/jetstream.nats.io/v1beta2/Stream";
+import { Consumer } from "@kubernetes-models/nats/jetstream.nats.io/v1beta2/Consumer";
+
+// Create a new NATS JetStream stream
+const myStream = new Stream({
+  metadata: { name: "mystream" },
+  spec: {
+    name: "mystream",
+    subjects: ["orders.*"],
+    storage: "memory",
+    maxAge: "1h"
+  }
+});
+
+// Create a new NATS JetStream consumer
+const myPushConsumer = new Stream({
+  metadata: { name: "my-push-consumer" },
+  spec: {
+    streamName: "mystream",
+    durableName: "my-push-consumer",
+    deliverSubject: "my-push-consumer.orders",
+    deliverPolicy: "last",
+    ackPolicy: "none",
+    replayPolicy: "instant"
+  }
+});
+
+// Create a new NATS JetStream consumer
+const myPullConsumer = new Stream({
```

Comment:
```suggestion
const myPullConsumer = new Consumer({
```

Reply:
Thanks, fixed in https://github.com/tommy351/kubernetes-models-ts/pull/150/commits/f4943fb5b6d362ae58b4a7eaae847123bdfd5e91.